about.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- Global site tag (gtag.js) - Google Analytics
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-YJ2WSX508J"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+      gtag("config", "G-YJ2WSX508J");
+    </script> -->
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="badakhsh portfolio, portfolio" />
+
+    <!-- Links -->
+    <link href="style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="project icon"
+      href="./assets/logos/Logo.png"
+      type="image/x-icon"
+    />
+
+    <!-- Script
+    <script src="app.js"></script> -->
+
+    <!-- Project Title -->
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <header>
+      <!--Header section-->
+      <div class="header">
+        <div class="quicklink social">
+          <a href="#"
+            ><img src="assets/logos/linkedinicon.png" alt="linked In Icon"
+          /></a>
+        </div>
+        <div class="quicklink social">
+          <a href="#"
+            ><img src="assets/logos/Twitter.png" alt="twitter logo"
+          /></a>
+        </div>
+        <div class="quicklink">
+          <p><a href="#">English</a></p>
+        </div>
+        <div class="quicklink">
+          <p><a href="#">My Page</a></p>
+        </div>
+        <div class="quicklink">
+          <p><a href="#">Logout</a></p>
+        </div>
+      </div>
+
+      <!-- Navbar Section -->
+      <div class="humberger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+      </div>
+      <div id="navbar" class="navbar">
+        <div class="logo">
+          <a href="index.html"
+            ><img src="assets/logos/Logo.png" alt="Page logo"
+          /></a>
+        </div>
+        <ul>
+          <li class="links"><a href="about.html">About </a></li>
```

Comment:
- [ ] As the previous reviewer said, The user cannot navigate from the about page to the home page. It would be better to add a "Home" link to the navbar in the `about` page so the user can navigate to the home page.

Reply:
👍🏽 Great job implementing this. I was able to navigate to your Homepage from the About page. (Desktop)